config/runtimes/kustomization.yaml

```diff
@@ -21,7 +21,7 @@ images:
 
   - name: mlserver
     newName: docker.io/seldonio/mlserver
-    newTag: 1.0.0
+    newTag: 1.5.0
```

Comment:
We can not use the [`seldonio/mlserver`](https://hub.docker.com/r/seldonio/mlflowserver) 1.0.0 since that has been removed from dockerhub.

```sh
NAME                                                              READY   STATUS             RESTARTS   AGE
isvc-sklearn-mms-v2-predictor-default-00001-deployment-757xrdqz   0/3     ImagePullBackOff   0          31m
isvc-transformer-predictor-default-00001-deployment-965899dqjkx   2/2     Terminating        0          2m1s
isvc-transformer-transformer-default-00001-deployment-7877jr4wl   2/2     Terminating        0          2m1s
isvc-xgboost-mms-v2-predictor-default-00001-deployment-dbdxc2fv   0/3     ImagePullBackOff   0          17m
Name:             isvc-sklearn-mms-v2-predictor-default-00001-deployment-757xrdqz
Namespace:        kserve-ci-e2e-test
...
Events:
  Type     Reason     Age                  From               Message
  ----     ------     ----                 ----               -------
  Normal   Scheduled  31m                  default-scheduler  Successfully assigned kserve-ci-e2e-test/isvc-sklearn-mms-v2-predictor-default-00001-deployment-757xrdqz to fv-az403-410
  Normal   Pulled     29m                  kubelet            Container image "gcr.io/knative-releases/knative.dev/serving/cmd/queue@sha256:a40f6e84de1a0d145d27084a94cc7fa221159e75cafde7d332ac8f4f0aed58fb" already present on machine
  Warning  Failed     29m                  kubelet            Failed to pull image "docker.io/seldonio/mlserver:1.0.0": rpc error: code = Unknown desc = context deadline exceeded
  Warning  Failed     29m                  kubelet            Error: ErrImagePull
  Normal   Started    29m                  kubelet            Started container queue-proxy
  Normal   Created    29m                  kubelet            Created container queue-proxy
  Normal   Pulled     29m                  kubelet            Container image "kserve/agent:d9f4de88476[82](https://github.com/kserve/kserve/actions/runs/3669245616/jobs/6203309932#step:11:83)06f1ec20195ea[88](https://github.com/kserve/kserve/actions/runs/3669245616/jobs/6203309932#step:11:89)dd5fb36b159d" already present on machine
  Normal   Created    29m                  kubelet            Created container agent
  Normal   Started    29m                  kubelet            Started container agent
  Warning  Failed     29m (x3 over 29m)    kubelet            Error: ImagePullBackOff
  Normal   Pulling    29m (x2 over 31m)    kubelet            Pulling image "docker.io/seldonio/mlserver:1.0.0"
  Warning  Unhealthy  21m (x39 over 29m)   kubelet            Readiness probe failed: Get "http://172.17.0.19:[90](https://github.com/kserve/kserve/actions/runs/3669245616/jobs/6203309932#step:11:91)81/": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
  Normal   BackOff    11m (x34 over 29m)   kubelet            Back-off pulling image "docker.io/seldonio/mlserver:1.0.0"
  Warning  Unhealthy  [94](https://github.com/kserve/kserve/actions/runs/3669245616/jobs/6203309932#step:11:95)s (x214 over 29m)  kubelet            Readiness probe failed: Get "http://[172](https://github.com/kserve/kserve/actions/runs/3669245616/jobs/6203309932#step:11:173).17.0.19:8012/": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
```

https://github.com/kserve/kserve/actions/runs/3669245616/jobs/6203309932#step:11:67

Reply:
I created an issue.
#2598

Reply:
My misunderstand. There is seldonio/mlserver:1.0.0 in dockerhub.